e2e/rbd.go

```diff
@@ -57,6 +57,9 @@ var (
 	appBlockSmartClonePath = rbdExamplePath + "block-pod-clone.yaml"
 	snapshotPath           = rbdExamplePath + "snapshot.yaml"
 	defaultCloneCount      = 10
+
+	kernelRelease = ""
```

Comment:
kernel release does not need to  be a global variable, I think?

Reply:
We run all the rbd tests in one go right?
What is the problem with reusing the kernel release?

Reply:
I think I got you!